client/lib/products-values/constants.js

```diff
@@ -21,6 +21,8 @@ export const PRODUCT_WPCOM_SEARCH = 'wpcom_search';
 export const PRODUCT_WPCOM_SEARCH_MONTHLY = 'wpcom_search_monthly';
 export const PRODUCT_JETPACK_SCAN = 'jetpack_scan';
 export const PRODUCT_JETPACK_SCAN_MONTHLY = 'jetpack_scan_monthly';
+export const PRODUCT_JETPACK_ANTISPAM = 'jetpack_antispam';
+export const PRODUCT_JETPACK_ANTISPAM_MONTHLY = 'jetpack_antispam_monthly';
```

Comment:
Since we used `jetpack_anti_spam` as product_slug, I think this has to change accordingly. Should we rename those constants as well?

Reply:
+1

Reply:
Resolved in 9b90064b9c4ee3c3e26d380a29b4051ebc1328b2.